schemas/schema.json

```diff
@@ -0,0 +1,151 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Air--MissileStrike",
+  "type": "object",
+  "properties": {
+    "Reference": {
+      "type": "string",
+      "title": "Reference (locked)",
+      "readOnly": true
+    },
+    "actionsOn": {
+      "type": "string",
+      "title": "Actions on"
+    },
+    "activity": {
+      "type": "string",
+      "enum": ["transit", "kinetic", "asymmetric"],
+      "title": "Activity",
+      "readOnly": true
+    },
+    "activityCard": {
+      "type": "string",
+      "enum": [
+        "None",
+        "FR Cyper Op 12 - Green Drain",
+        "DE Cyper Op 14 - Brain Smash",
+        "GR Space Op 16 - Ruby Spaniel",
+        "SP Info Op 18 - Orange Drawn"
+      ],
+      "title": "Associated Cyber/Space Activity Ref#"
+    },
+    "activityDescription": {
+      "type": "string",
+      "title": "Activity Description"
+    },
+    "emconState": {
+      "type": "string",
+      "enum": [
+        "Normal Operations",
+        "Electronic Silence",
+        "Radio Silence",
+        "No restrictions"
+      ],
+      "title": "Emcon State"
+    },
+    "endDate": {
+      "type": "string",
+      "format": "date-time",
+      "title": "End Date"
+    },
+    "intent": {
+      "type": "string",
+      "title": "Intent"
+    },
+    "location": {
+      "type": "string",
+      "title": "Location",
+      "readOnly": true
+    },
+    "otherAssets": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "asset": {
+            "type": "string",
+            "enum": ["ta", "tb", "tc", "pa"],
+            "title": "Target"
+          },
+          "missileType": {
+            "type": "string",
+            "enum": [
+              "###Ballistic",
+              "SRBM",
+              "MRBM",
+              "IRBM",
+              "###Cruise",
+              "Standard Cruise",
+              "Low Obs Cruise",
+              "###UAV",
+              "Propellor OWA UAV",
+              "Jet OWA UAV"
+            ],
+            "title": "Missile Type"
+          },
+          "number": {
+            "type": "number",
+            "title": "Quantity"
+          }
+        },
+        "required": ["asset", "number"]
+      },
+      "title": "Targets"
+    },
+    "ownAssets": {
```

Comment:
thanks for tidying up / converting this @Talented0217  - but could you include the definition for ownAssets?

Reply:
Sure

Reply:
I've fixed it